public/app/views/shared/_footer.html.erb

```diff
@@ -4,7 +4,7 @@
        <p class="footer-items"><a href="<%= AppConfig[:frontend_proxy_url] %>">Staff Interface</a>
          | Visit <a href="http://archivesspace.org">ArchivesSpace.org</a>
```

Comment:
If you want to punt, the rest of this looks totally fine and I'll be happy to merge it in. Thanks.

Reply:
I was under the impression that the idea of the ArchivesSpace.org link was specifically to go to ArchivesSpace, rather than to the main url?

Reply:
Oh sorry I misspoke, to clarify, when I said main url, I didn't mean the application url, but rather the archivesspace url. I'm asking since the feedback url value seems to be `http://archivesspace.org/feedback` [here](https://github.com/archivesspace/archivesspace/pull/1075/files#diff-84522df7df17f17c875fa36af2c78cacR7)

Reply:
@eshadatta I don't understand what you are asking for here. Can you provide an example?

Reply:
@lmcglohon ,

In the config/config-defaults file, similar to the feedback url key, have something like this:
```
# URL to direct link to archivesspace
# You can remove this from the footer by making the value blank.
AppConfig[:archivesspace_url] = "http://archivesspace.org"
```

Then the `_footer.html.erb` file will have the following code:
```
Visit <a href='<%= AppConfig[:archivesspace_url]%>'><%= AppConfig[:archivesspace_url]%></a>
```
The feedback url's value is `archivesspace.org/feedback`. I'm just saying that we could have `archivesspace.org` in a key as well instead of hard-coding that. If it's too nit-pickey, I'm totally fine with ignoring this feedback and merging the code as is, since it's perfectly alright.

Reply:
@eshadatta I see what you mean now. I think it is ok to go ahead and merge Bobbi's PR since it fixes the issue at hand. If you would like to create a Github pull request with your suggested change in it, that would be great!

Reply:
@lmcglohon , will do!